### ISSUE TITLE
ValueConverter registration in WPF

### ISSUE THREAD

**Issue by indazoo** [MvvmCross/MvvmCross#1212]
Hi
I tried Tibet binding in WPF and found some errors in the documentation and additionally request a code change.

1. Small Typo:
https://github.com/MvvmCross/MvvmCross/wiki/Value-Converters#valueconverters-and-fallbackvalues
==> "FalllbackValue"  has three "l" letters  (this occurs two times)

2. Missing text
https://github.com/MvvmCross/MvvmCross/wiki/Value-Converters#tibet-valuecombiners
The following sentence makes no sense
"If you do want to add is used by default on the , then a ValueCombiner class named FooValueCombiner will be registered under the name Foo."

3. Please, please include in the Tibet samples (Windows, WPF) a sample where two properties are being set. I had to search in the videos to find that a semicolon is used for that. Thanks

4. Misleading explanation
In WPF the automatic registration of the converters does not work. For me and also for others:
http://stackoverflow.com/questions/30488732/mvvmcross-wpf-value-converter-with-tibet-binding-is-not-converting
In the docs https://github.com/MvvmCross/MvvmCross/wiki/Value-Converters#using-value-converters-in-windows-tibet-binding is the "magic" class and its import into XAML for Windows explained.
As a newcomer, you do that and it does not work. Some hair pulling later you find that you have to do the registration by oneself. Well, please include some text stating that in WPF it will not work and you have to do some manual registering and the Import block is not needed or even better please include that registration mechanism in MvxWpfSetup class as described in the StackOverflow question so the doc is correct and must not be changed.

Thanks!

**Comment by kjeremy** [comment]
A pull request would be great!

**Comment by Cheesebaron** [comment]
You are right, there are errors in that documentation.

WPF, just like UWP, WindowsPhone and WindowsStore project need their ValueConverters registered in the App.xaml file. You need to do that for all the converters you expect to reference through a StaticReference anywhere in your XAML pages. In order to register a `MvxValueConverter` in the App.xaml file, the type needs to be converted to something that the App.xaml file understands. So all MvxValueConverters you are going to register there need to be wrapped in `MvxNativeValueConverter`. Something in the likes of:

```
public class NativeVisibilityConverter : 
    MvxNativeValueConverter<MvxVisibilityValueConverter>{}
```

Then you need an equivalent entry in your ResourceDictionary in your App.xaml file:

```
<nativeConverters:NativeVisibilityConverter x:Name="Visibility" />
```

**Comment by indazoo** [comment]
Some confusion on my side now:
The docs state that for tibet binding the manual registration is not needed:
"When Tibet binding is used, then Value Converters can be accessed by name - exactly as in Droid and Touch binding - without the above native Xaml wrapping."

If I knew what the goal is I would do the pull request. But let's be honest. I just start scratching the surface of MvvmCross.

**Comment by Cheesebaron** [comment]
I don't think that is correct. Not if you are using the binding expressions provided by XAML. If you are using BindingEx on the other hand, that might be an entirely other case.

**Comment by indazoo** [comment]
More head scratching: I thought for tibet binding using bindingEx is mandatory.

**Comment by Cheesebaron** [comment]
Not on Android or iOS.

**Comment by indazoo** [comment]
Thanks for that hint "Not on Android or iOS". I might need it in the future. At the moment I have a Windows App and think your Tibet binding is cool :-)

**Comment by nickrandolph** [comment]
@indazoo is there still work to be done on this?

**Comment by indazoo** [comment]
Hi, honestly I stopped working on this. Too much things didn't worked for me.
